### PR TITLE
Fix issue 2419

### PR DESCRIPTION
**Pull request by jomadoye**
Fixes issue #2419 
screen shot 2018-09-09 at 11 29 35 pm

### CODE REVIEW

**Comment by maxceem** [comment]
@jomadoye Please, resolve lint errors before this PR can be merged.

Also, FYI I've changed the base branch for this PR from `dev` to `cf12` as all PR has to be made against `cf12` branch. Thanks.

**Comment by jomadoye** [comment]
Alright, I am working on this. Thanks.

**Comment by jomadoye** [comment]
This has been fixed @maxceem

File: src/projects/detail/containers/ProjectPlanContainer.jsx [hunk 1]
```diff
@@ -120,7 +117,6 @@ class ProjectPlanContainer extends React.Component {
         <TwoColsLayout.Content>
           {visiblePhases && visiblePhases.length > 0 ? (
             [
-              activePhases.length > 0 && <ProjectPlanProgress phases={visiblePhases} project={project} productsTimelines={productsTimelines} key="progress" />,
               <ProjectStages
```
Comment (maxceem) [inline src/projects/detail/containers/ProjectPlanContainer.jsx]: Sorry that I didn't mention it from the beginning, but there is one more small thing. As `<ProjectStages ...` is the only one component left in the array now, we don't need to wrap it into the array anymore. Also we don't have to define a `key` property now as there is no array anymore https://github.com/appirio-tech/connect-app/blob/d583a8bf86e371fcac8121ccc2cb9a333e369ae6/src/projects/detail/containers/ProjectPlanContainer.jsx#L125

**Comment by jomadoye** [comment]
@maxceem This has been uopdated

**Pull request merged**